app/templates/snips/courseElements/editCourse.html

```diff
@@ -6,11 +6,20 @@
     <form method="post" action="\editcourse\{{currentTerm}}\{{course.prefix.prefix}}\{{page}}">
       <div class="modal-content">
         <div class="modal-header">
-          <a href="/courses/{{currentTerm}}/{{course.prefix.prefix}}">
-            <button type="button" class="close" aria-label="Close">
-              <span aria-hidden="true">&times;</span>
-            </button>
-          </a>
+          {# DIFFERENT URL'S DEPENDING ON THE PAGE #}
+          {% if page == "courses" %}
+            <a href="/courses/{{currentTerm}}/{{course.prefix.prefix}}">
+              <button type="button" class="close" aria-label="Close">
+              <span>&times;</span>
+              </button>
+            </a>
+          {% else %}
+            <a href="/courseManagement/{{page}}/{{currentTerm}}">
+              <button type="button" class="close" aria-label="Close">
+              <span>&times;</span>
+              </button>
+            </a>
+          {% endif %}
```

Comment:
The only thing that is different between these cases is the url, so we shouldn't repeat all of the other html.